apps/www/content/docs/api/slate-utils.mdx

```diff
@@ -256,14 +256,14 @@ Returns the point next to the void.
 
 ### getPreviousBlockById
 
-Finds the block before a block by ID. If not found, it finds the first block by ID and returns **`[null, its previous path]`**.
+Finds the block before a block by id. If not found, it finds the first block by id and returns **`[null, its previous path]`**.
 
 <APIParameters>
   <APIItem name="editor" type="TEditor">
     The editor where to find the previous block.
   </APIItem>
   <APIItem name="id" type="string">
-    The ID of the block.
+    The id of the block.
```

Comment:
What's the rationale behind changing "ID" to "id"? Usually, I'd only expect to see "id" in lowercase when it's formatted as code (i.e. `id`).

Reply:
Was not sure. Makes more sense now :) Fixed.